frontend/src/components/public_group/PublicGroupExpensesAndActivity.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import PublicGroupDonations from './PublicGroupDonations';
+import PublicGroupExpenses from './PublicGroupExpenses';
+
+export default class PublicGroupExpensesAndActivity extends Component {
```

Comment:
Nothing, for some reason I didn't realize I could do that, I will use a pure functional component